cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -913,6 +913,37 @@ and a constraint can be one of::
     }
 
 
+If the current app is a :ref:`CDAP Pipeline <cdap-pipelines>`, and the schedule contains
```

Comment:
these arguments are available only for that particular app? Not any schedule program (regardless of its containing app)?

Reply:
right, it's only available in CDAP pipeline, since the syntax is defined by each app